dashboard/src/shared/schema.ts

```diff
@@ -0,0 +1,17 @@
+import { IBasicFormParam } from "./types";
+
+export function retrieveBasicFormParams(schema: any) {
+  // TBD
```

Comment:
Thanks for leaving this out of the PR to make the diff more manageable. I'll have a quick break before reviewing the next :)